app/controllers/attachments_controller.rb

```diff
@@ -1,49 +0,0 @@
-class AttachmentsController < BaseAttachmentsController
-  include PublicDocumentRoutesHelper
-
-  def show
-    unless user_signed_in?
```

Comment:
Something @barrucadu spotted, we need to either keep this redirection here or move it into Nginx otherwise we'll break all the old asset URLs are that `www.gov.uk/government/uploads/...`.

Reply:
I think this case will be handled by https://github.com/alphagov/whitehall/blob/master/app/controllers/asset_manager_redirect_controller.rb